scripts/sync-dashboards

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+from urllib.request import urlretrieve
+
+# sync the dashboards from https://opendev.org/openstack/sunbeam-charms/src/branch/main/charms/openstack-exporter-k8s/src/grafana_dashboards
+BASE_URL = "https://opendev.org/openstack/sunbeam-charms/raw/branch/main/charms/openstack-exporter-k8s/src/grafana_dashboards/"
+BASE_LOCAL_PATH = "./src/grafana_dashboards/"
+DASHBOARDS = [
+    "hypervisor-openstack-exporter-dashboard.json",
+    "openstack-exporter.json",
+    "overview-openstack-exporter-dashboard.json",
+]
+
+
+def sync_dashboard(url: str, local_path: str, check_mode: bool) -> bool:
+    """Sync dashboard from url.
+
+    If check_mode is True, don't actually update the file.
+    Return True if the dashboard changed (or would be changed).
+    """
+    new_dashboard_path, _ = urlretrieve(url)
+
+    changed = False
+    if os.path.exists(local_path):
```

Comment:
I think we should not check the existence of `local_path`? since the remote is the source of truth. I want to confirm, what is the behavior you are expecting in the following case:

1. if there are new or missing files from "REMOTE", should we report difference?

Reply:
Yeah, they will not happen often or even won't happen, but still nice to do it if it's not difficult though (I don't want to over-complicate this script), otherwise it's not "sync-dashboards". I was more expecting `rsycn ... --delete <remote>  <src>` like behaviour

Reply:
👌🏼